### DISCUSSION TITLE
Question about copilot and the student pack

### DISCUSSION THREAD

**Discussion by Hiviexd** [community/community#144777] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I recently was paying for Copilot, then I just got approved for the student pack which advertises free copilot access, yet when I go to my billings summary, it's still showing that the monthly $10 payment is still active and will bill me on the next cycle.

I'm wondering what do I need to do to make sure it's using the student pack plan, thanks in advance.

P.S. I'd like to confirm that I got access to the other benefits of the pack.

### Screenshots

![image](https://github.com/user-attachments/assets/b44aa1ba-d79e-4888-aa6f-81f87d29568c)

![image](https://github.com/user-attachments/assets/49c8b3ec-cdd1-4e52-8adb-a0f514efe1e5)

**Comment by davevad93** [comment, marked as answer]
Hi @Hiviexd ,

If your status label is purple and states "`Congrats! Your request for @Hiviexd was approved and your benefits are now available.`", that means you have access to the `Student Developer Pack` offers.

As I can see from your first screenshot, your student benefits coupon is applied and expires in 2 years, which is exactly the duration of GitHub Education benefits, after which you have to re-verify your student status in order to renew them.

![0000](https://github.com/user-attachments/assets/3bb54f64-8686-44f9-b672-743da0decae3)

Since you now have access to `Copilot` for free, you shouldn't be charged on Dec 4th, but if you want to be absolutely sure, contact support by opening a ticket [here](https://support.github.com/contact/account), select `Billing, payments or receipts` and ask them.

**Reply by Hiviexd** [reply]
My status label is just as you've described.

![image](https://github.com/user-attachments/assets/095fc86a-860c-4963-b0a2-f5d0af825af9)

I'm mainly worried that i could be billed on Dec 4th based on what the billing page is telling me, so I'll send a support ticket just in case.

Thanks for the assistance!
